src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -1111,4 +1111,8 @@ For more information, visit https://docs.nuget.org/docs/reference/command-line-r
   <data name="SpecValidation_OriginalTargetFrameworksMustMatchAliases" xml:space="preserve">
     <value>The original target frameworks value must match the aliases. Original target frameworks: {0}, aliases: {1}.</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="Error_PlatformVersionNotPresent" xml:space="preserve">
+    <value>TargetFramework '{0}' has a platform '{1}', but a platform version could not be inferred. Specify the platform version explicitly.</value>
```

Comment:
I wouldn't use `TargetFramework`, so that we don't suggest that the TargetFramework in particular is an issue.

Reply:
I don't necessarily have a good suggestion here, but the message provides non-actionable (maybe even incorrect suggestion to the customer). The expectation is that the SDK does this, unless you have your own SDK.